api/envoy/extensions/filters/http/response_map/v3/response_map.proto

```diff
@@ -81,3 +81,14 @@ message ResponseMap {
   //
   config.core.v3.SubstitutionFormatString body_format = 2;
 }
+
+// Extra settings on a per virtualhost/route/weighted-cluster level.
+message ResponseMapPerRoute {
+  oneof override {
+    option (validate.required) = true;
+
+    // Disable the response map filter for this particular vhost or route.
+    // If disabled is specified in multiple per-filter-configs, the most specific one will be used.
+    bool disabled = 1 [(validate.rules).bool = {const: true}];
```

Comment:
I see those `validate.` annotations on a few other filters (but not the `grpc_http1_reverse_bridge` filter that you mentioned cribbing the approach from), so I guess I trust that they're correct, but I'm flummoxed as to interpreting what they actually mean.

Reply:
i tested this and it means disabled must be a bool equal to true. false is not valid.

Reply:
Why shouldn't false be valid? And if it's `validate.required`, then how do you omit it (I guess you just leave out the entire `ResponseMapPerRoute`?)?

Reply:
i think the idea is to omit if you want it to be not disabled. it's opinionated for sure.